sale_automatic_workflow/__manifest__.py

```diff
@@ -5,15 +5,15 @@
 
 {
     "name": "Sale Automatic Workflow",
-    "version": "14.0.1.1.2",
+    "version": "14.0.1.2.0",
     "category": "Sales Management",
     "license": "AGPL-3",
     "author": "Akretion, "
     "Camptocamp, "
     "Sodexis, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
-    "depends": ["sale_stock", "sales_team"],
+    "depends": ["sale_stock", "sales_team", "sale_delivery_state"],
```

Comment:
You can't add such new dependency in a stable module.

Reply:
The only way I see around this is that for v13 and v14 we add another module w/ auto_install=True that overrides sale_automatic_workflow behavior.

Reply:
Hi @pedrobaeza I removed the dependency and added a glue module (with auto_install=True)